examples/README.md

```diff
@@ -12,6 +12,130 @@
+  - [Tailing a Single File](#tailing-a-single-file)
+- [Windows Specific Processors](#windows-specific-processors)
+  - [Consuming Windows Event Logs](#consuming-windows-event-logs)
+  - [Reading System Performance Data](#reading-system-performance-data)
+- [HTTP Operations](#http-operations)
+  - [HTTP POST Invocation](#http-post-invocation)
+- [Site to Site Operations](#site-to-site-operations)
+  - [Transfer Data to Remote Nifi Instance](#transfer-data-to-remote-nifi-instance)
+  - [Site-2-Site Bi-directional Configuration](#site-2-site-bi-directional-configuration)
+- [Kafka Operations](#kafka-operations)
+  - [Publish Message to Kafka Broker](#publish-message-to-kafka-broker)
+  - [Publish Message to Kafka Broker Through SSL](#publish-message-to-kafka-broker-through-ssl)
+  - [Consume Messages from Kafka](#consume-messages-from-kafka)
+- [Public Cloud Operations](#public-cloud-operations)
+  - [Upload Blob to Azure Storage](#upload-blob-to-azure-storage)
+  - [Put Object in AWS S3 Bucket](#put-object-in-aws-s3-bucket)
+  - [List and Fetch Content from AWS S3 Bucket](#list-and-fetch-content-from-aws-s3-bucket)
+- [SQL Operations](#sql-operations)
+  - [Query Database Table](#query-database-table)
+- [System Log Operations](#system-log-operations)
+  - [Consume Systemd-Journald System Journal Messages](#consume-systemd-journald-system-journal-messages)
+
+## Filesystem Operations
+
+### Getting File Data and Putting It in an Output Directory
+
+Using the [getfile_putfile_config.yml](getfile_putfile_config.yml) flow configuration MiNiFi gets all files of minimum 1MB size from the `/tmp/getfile_dir` directory and puts them in the `/tmp/out_dir` output directory.
+
+The flow: GetFile &#10132; success &#10132; PutFile
+
+### Tailing a Single File
+
+Using the [tailfile_config.yml](tailfile_config.yml) flow configuration MiNiFi tails a single file `/tmp/test_file.log` and creates flowfiles from every single line, then logs the flowfile attributes.
+
+The flow: TailFile &#10132; success &#10132; LogAttribute
+
+## Windows Specific Processors
+
+### Consuming Windows Event Logs
+
+Using the [cwel_config.yml](cwel_config.yml) flow configuration MiNiFi queries all Windows system events and puts them to the `C:\temp\` directory in flattened JSON format.
+
+The flow: ConsumeWindowsEventLog &#10132; PutFile
+
+## Linux Specific Processors
+
+### Consume Systemd-Journald System Journal Messages
+
+Using the [consumejournald_config.yml](consumejournald_config.yml) flow configuration MiNiFi reads systemd-journald journal messages and logs them on `info` level.
+
+The flow: ConsumeJournald &#10132; LogAttribute
+
+### Reading System Performance Data
+
+Using the [pdh_config.yml](pdh_config.yml) flow configuration MiNiFi reads CPU and Disk performance data through Windows' Performance Data Helper (PDH) component and puts the data to the `C:\temp\` directory in a compact JSON format.
+
+The flow: PerformanceDataMonitor &#10132; PutFile
+
```

Comment:
Could you move this up to the Windows-specific section? It's now under Linux-specific processors.

Reply:
Good catch, fixed in d02f0d09ded60838efd5498b9627bc4e841e1db5